src/components/RepoInputer.vue

```diff
@@ -82,7 +82,7 @@
 
 <script lang="ts" setup>
 import { computed, onMounted, reactive, ref, watch } from "vue";
-import { first } from "lodash-es";
+import { first } from "lodash";
```

Comment:
Have you tested if this works? Seems lodash has renamed it to [_.head](https://lodash.com/docs/4.17.15#head). I'm not sure if `first` still works. Change it to `head` may be better.

Reply:
SGTM, updated.